tests/lang_callable/template/template_usage_substitution_typed_params.nim

```diff
@@ -0,0 +1,32 @@
+discard """
+  description: '''
+Templates `typed` and non-AST parameters do not replace identifiers in new
+symbol definition positions. Meaning a template parameter that is not `untyped`
```

Comment:
```suggestion
Except for `untyped`, template parameters do not replace identifiers in new
symbol definition positions. Meaning a template parameter that is not `untyped`
```
I think this is bit easier to understand. It's a soft suggestion, feel free to ignore.

Reply:
Yup, definitely easier to read. Thanks.